packages/react-components/src/VoteActions.tsx

```diff
@@ -28,11 +32,17 @@ function VoteActions ({ accountId, aye, className = '', isDisabled, onClick, par
     >
       <TxButton
         accountId={accountId}
+        extrinsic={extrinsic}
         icon='check'
         isDisabled={!accountId || isDisabled}
         label={aye
-          ? t<string>('Vote Aye')
-          : t<string>('Vote Nay')}
+          ? isClosing
+            ? t<string>('Vote Aye and Close')
```

Comment:
Not the nicest button but I thought it's always better to explicitly tell users what this batch is supposed to do.